docs-md/tutorials/embedded-connect.md

```diff
@@ -0,0 +1,409 @@
+tagline: Run Kafka Connect embedded within ksqlDB
+description: Learn how to use ksqlDB with embedded Connect to integrate with external data sources and sinks
+keywords: ksqlDB, connect, PostgreSQL, jdbc
+---
+
+Overview
+==============
+
+This tutorial will demonstrate how to integrate ksqlDB with an external data
+source to power a simple ride sharing app. Our external source will be a
+PostgreSQL database containing relatively static data describing each driver’s
+vehicle. By combining this human-friendly static data with a continuous stream
+of computer-friendly driver and rider location events, we derive an enriched
+output stream that the ride sharing app may use to facilitate a rendezvous in
+real time.
+
+1. Get ksqlDB
+--------------
+
+Since ksqlDB runs natively on {{ site.aktm }}, you need a running {{ site.ak }}
+installation that ksqlDB is configured to use. The following docker-compose
+files run everything for you via Docker, including ksqlDB running
+[Kafka Connect](https://docs.confluent.io/current/connect/index.html) in
+embedded mode. Embedded Connect enables you to leverage the power of
+{{ site.kconnect }} without having to manage a separate {{ site.kconnect }}
+cluster, because ksqlDB manages one for you. Also, this tutorial use PostgreSQL
+as an external datastore to integrate with ksqlDB.
+
+In an empty local working directory, copy and paste the following
+`docker-compose` content into a file named `docker-compose.yml`. You will
+create and add a number of other files to this directory during this tutorial.
+
+```yaml
+---
+version: '2'
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.3.1
+    hostname: zookeeper
+    container_name: zookeeper
+    ports:
+      - "2181:2181"
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+
+  broker:
+    image: confluentinc/cp-enterprise-kafka:5.3.1
+    hostname: broker
+    container_name: broker
+    depends_on:
+      - zookeeper
+    ports:
+      - "29092:29092"
+    environment:
+      KAFKA_BROKER_ID: 1
+      KAFKA_ZOOKEEPER_CONNECT: 'zookeeper:2181'
+      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://broker:9092,PLAINTEXT_HOST://localhost:29092
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
+
+  ksqldb-server:
+    image: confluentinc/ksqldb-server:0.6.0
+    hostname: ksqldb-server
+    container_name: ksqldb-server
+    depends_on:
+      - broker
+    ports:
+      - "8088:8088"
+    environment:
+      KSQL_LISTENERS: http://0.0.0.0:8088
+      KSQL_BOOTSTRAP_SERVERS: broker:9092
+      KSQL_KSQL_LOGGING_PROCESSING_STREAM_AUTO_CREATE: "true"
+      KSQL_KSQL_LOGGING_PROCESSING_TOPIC_AUTO_CREATE: "true"
+      KSQL_KSQL_CONNECT_WORKER_CONFIG: "/connect/connect.properties"
+    volumes:
+      - ./confluentinc-kafka-connect-jdbc-5.3.1:/usr/share/kafka/plugins/jdbc
```

Comment:
5.3.2 is the current JDBC connector download version on Confluent Hub

Reply:
Updated